packages/desktop/src/services/FileDownloadServiceElectron.ts

```diff
@@ -256,7 +358,7 @@ export default class FileDownloadServiceElectron
                             })
                         );
                         await this.deleteArtifact(outFilePath);
-                    } catch (err) {
+                    } catch (err: unknown) {
```

Comment:
Do we need this?
```suggestion
                    } catch (err) {
```

Reply:
[cfdf9b4](https://github.com/AllenInstitute/biofile-finder/pull/204/commits/cfdf9b40628c86077ac4ad73f158267bde3b5b49)